README.md

```diff
@@ -1,7 +1,6 @@
 # RepoSense
 
 [![Build Status](https://github.com/reposense/RepoSense/actions/workflows/integration.yml/badge.svg)](https://github.com/reposense/RepoSense/actions/workflows/integration.yml)
-[![Build status](https://ci.appveyor.com/api/projects/status/gsbkj5qby3pjd6nw/branch/master?svg=true)](https://ci.appveyor.com/project/eugenepeh/reposense/branch/master)
```

Comment:
Not sure if this is in the scope of this PR?

Reply:
Yea it's tenuously related (since the yml removal raised awareness of this dependence). I can open another pr if we want to separate these changes.

Reply:
Will such a PR only consist of this change?

Reply:
I think I'd prefer to add it to the scope of this one, since this would be the only change